src/xSTIR/cSTIR/include/sirf/STIR/stir_x.h

```diff
@@ -958,6 +958,13 @@ The actual algorithm is described in
 		}
```

Comment:
Perhaps more methods need to be exposed by this accessor class (i.e. kappa)?

Reply:
public members in the STIR class are inherited. It would make sense to override `set/get_kappa_sptr` could be overridden by SIRF versions that use `SIRFImageData`

Reply:
Great thanks.